src/main/java/org/breedinginsight/api/v1/controller/AccessibilityController.java

```diff
@@ -0,0 +1,65 @@
+package org.breedinginsight.api.v1.controller;
```

Comment:
While you were working on this we added the apache license headers to our source files (see files in the latest develop for formatting). Could you add the header to the new files you added here that don't have them. Also not sure if you can setup emacs to automatically add them to new files but that's what I do with intellij.

Reply:
Thanks, I added the headers.